simulariumio/data_objects/model_meta_data.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Any
+import logging
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class ModelMetaData:
+    title: str
+    version: str
+    authors: str
+    description: str
+    doi: str
+    source_code_url: str
+    source_code_license_url: str
+    input_data_url: str
+    raw_output_data_url: str
+
+    def __init__(
+        self,
+        title: str = None,
+        version: str = None,
+        authors: str = None,
+        description: str = None,
+        doi: str = None,
+        source_code_url: str = None,
+        source_code_license_url: str = None,
+        input_data_url: str = None,
+        raw_output_data_url: str = None,
+    ):
+        """
+        This object holds metadata for a model that generates simulation trajectories
+        Parameters
+        ----------
+        title : str (optional)
+            Display title for this model
+        version : float (optional)
+            Version number of the model that produced this trajectory
+        authors : str (optional)
+            Modelers name(s)
+        description : str (optional)
+            Comments to display with the trajectories generated by this model
+        doi : str (optional)
+            The DOI of the publication accompanying this model
+        source_code_url : str (optional)
+            If the code that generated this model is posted publicly,
+            a link to the repository of source code
+        source_code_license_url : str (optional)
+            A link to the license for the source code
+        input_data_url : str (optional)
+            A link to any model configuration or parameter files posted publicly
+        raw_output_data_url : str (optional)
+            A link to any raw outputs from the source code posted publicly
+        """
+        self.title = title if title is not None else ""
+        self.version = version if version is not None else ""
+        self.authors = authors if authors is not None else ""
+        self.description = description if description is not None else ""
+        self.doi = doi if doi is not None else ""
+        self.source_code_url = source_code_url if source_code_url is not None else ""
+        self.source_code_license_url = (
+            source_code_license_url if source_code_license_url is not None else ""
+        )
+        self.input_data_url = input_data_url if input_data_url is not None else ""
+        self.raw_output_data_url = (
+            raw_output_data_url if raw_output_data_url is not None else ""
+        )
```

Comment:
I think you said you were going to do this already, but I think for these strings it would be cleaner to just use `""` as default args

Reply:
yep planning the fix that